src/content/configuration/output.md

```diff
@@ -616,7 +616,7 @@ Note that `[hash]` in this parameter will be replaced with an hash of the compil
 
 `boolean`
 
-Tell webpack to include comments in bundles with information about the contained modules. This option defaults to `false` and **should not** be used in production, but it's very useful in development when reading the generated code.
+Tell webpack to include comments in bundles with information about the contained modules. This option defaults to `true` in `development` and `false` in `production` [mode](/concepts/mode/) respectively. Please note that `output.pathinfo` **should not** be used in production, but it's very useful in development when reading the generated code.
```

Comment:
I think we should change _Tell_ for _Tells_. Also the final part of the sentence is a little confusing about using the property or enabling it.

I would change it to: _While the data this comments can provide is useful during development, it's recommended to be disabled for production._

I would even put it in a warning block `W>`

Reply:
rephrasing this a big so it explicitly states it shouldnt be used in prod:

```
W> While the data this comments can provide is very useful in development when reading the generated code, it **should not** be used in production.
```

**when reading the generated code** - should give a clue to the reader what exactly this gives